app/controllers/locations/floors_controller.rb

```diff
@@ -0,0 +1,62 @@
+require_dependency "locations/application_controller"
+
+module Locations
+  class FloorsController < ApplicationController
+    before_action :set_floor, only: [:show, :edit, :update, :destroy]
+
+    # GET /floors
+    def index
+      @floors = Floor.all
+    end
+
+    # GET /floors/1
+    def show
+    end
+
+    # GET /floors/new
+    def new
+      @floor = Floor.new(floor_params)
+    end
+
+    # GET /floors/1/edit
+    def edit
+    end
+
+    # POST /floors
+    def create
+      @floor = Floor.new(floor_params)
+
+      if @floor.save
+        redirect_to @floor, notice: 'Floor was successfully created.'
+      else
```

Comment:
Can you add: `flash.now[:error] = @floor.errors.full_messages` as the first part of this else clause?